docs/pages/powermanager_tests.md

```diff
@@ -0,0 +1,139 @@
+|3|[Testing Wakeup Source](#testing-wakeup-source)|Power manager should accept the supported wakeup sources and device should wakeup from standby/sleep modes using wakeup sources set by caller|
+|4|[Test Reset Functionality](#test-reset-functionality)|Power manager should reboot the device|
+
+## Emulator Requirements
+
+Boot configuration: Wakeup sources supported by the device
+
+Wakeup sources:
+
+- Voice Wakeup
+- Presence Detection wake up
+- Bluetooth wake up
+- Wifi wake up
+- `IR` wake up
+- Power key wake up
+- Timer wake up
+- `CEC` wake up
+- `LAN` wake up
+
+## Set And Get Power States
+
+|S.No.|Test Functionality|Description|HAL APIs|L2|L3|Control plane requirements|
+|-----|------------------|-----------|--------|--|--|--------------------------|
+|1|[Set and Get Power States](#set-and-get-power-states)|Set various power states and retrieve it for verification |PLAT_API_SetPowerState, PLAT_API_GetPowerState|Y|`NA`|`NA`|`NA`|
+
+### Test Startup Requirement - Set And Get Power States
+
+`NA`
+
+### Emulator Requirements - Set And Get Power States
+
+`NA`
+
+### Control Plane Requirements - Set And Get Power States
+
+`NA`
+
+## Set and Get Status of Wakeup Sources
+
+|S.No.|Test Functionality|Description|HAL APIs|L2|L3|Control plane requirements|
+|-----|------------------|-----------|--------|--|--|--------------------------|
+|1|[Set and Get Status of Wakeup Sources](#set-and-get-status-of-wakeup-sources)|Set status of various wakeup sources and retrieves status for verification |PLAT_API_SetWakeupSrc, PLAT_API_GetWakeupSrc|Y|`NA`|`NA`|`NA`|
+
+### Test Startup Requirement - Set and Get Status of Wakeup Sources
+
+`NA`
+
+### Emulator Requirements - Set and Get Status of Wakeup Sources
+
+[Emulator Requirements](#emulator-requirements)
+
+### Control Plane Requirements - Set and Get Status of Wakeup Sources
+
+`NA`
+
+## Testing Wakeup Source
+
+|S.No.|Test Functionality|Description|HAL APIs|L2|L3|Control plane requirements|
+|-----|------------------|-----------|--------|--|--|--------------------------|
+|1|[Testing Wakeup Source](#testing-wakeup-source)|Power manager should accept the supported wakeup sources and device should wakeup from standby/sleep modes using wakeup sources set by caller|PLAT_API_SetWakeupSrc, PLAT_DS_SetDeepSleep|`NA`|Y|Control plane to trigger the wakeup source|
+
+### Test Startup Requirement - Testing Wakeup Source
+
+`NA`
+
+### Emulator Requirements - Testing Wakeup Source
+
+[Emulator Requirements](#emulator-requirements)
+
+### Control Plane Requirements - Testing Wakeup Source
+
+Control plane to trigger the wakeup source supported by platform
+
+## Test Reset Functionality
+
+|S.No.|Test Functionality|Description|HAL APIs|L2|L3|Control plane requirements|
+|-----|------------------|-----------|--------|--|--|--------------------------|
+|1|[Test Reset Functionality](#test-reset-functionality)|Power manager should reboot the device|PLAT_Reset|`NA`|Y|`NA`||
```

Comment:
More description is required; the current description will not help generate the L2 low-level spec for ChatGPT.

Reply:
Increased the size of the description--admittedly I'm not entirely sure what else to add.